x-pack/plugins/security_solution/cypress/e2e/cases/attach_alert_to_case.cy.ts

```diff
@@ -19,6 +19,7 @@ import { ATTACH_ALERT_TO_CASE_BUTTON, ATTACH_TO_NEW_CASE_BUTTON } from '../../sc
 import { LOADING_INDICATOR } from '../../screens/security_header';
 
 const loadDetectionsPage = (role: ROLES) => {
+  login(role);
```

Comment:
instead of calling `login()` outside of the `loadDetectionsPages` we have moved it inside, so the user doesn't have to duplicate the passing a proper role in every place where it's being used
but you're right, from the test point of view, there are no changes